app/state_test.go

```diff
@@ -1330,6 +1330,32 @@ var stateTransitionsWithUpdateModulesTestCases []stateTransitionsWithUpdateModul
 		installOutcome: tests.SuccessfulRollback,
 	},
 
+	{
+		caseName: "Error in Download state, with rollback",
```

Comment:
I think this is a duplicate of an existing test case, isn't it?

Reply:
Well there is not any test for Download state with rollback, only with no rollback. If I understood one of the corrections from the meeting it was to add the test case with rollback enabled. But I could have understood wrong. It is not a duplicate since the rollback is different, should I then keep it or remove it?

Reply:
Ah, you're right, my bad. Please keep it! 👍